datumaro/components/extractor_tfds.py

```diff
@@ -102,20 +137,38 @@ def __call__(self, tfds_example: Any, item: DatasetItem) -> None:
 class _AddLabelFromClassLabelFeature:
     feature_name: str
 
-    def __call__(self, tfds_example: Any, item: DatasetItem) -> None:
+    def __call__(
+        self, tfds_example: Any, item: DatasetItem, state: namespace,
+    ) -> None:
         item.annotations.append(
             Label(tfds_example[self.feature_name].numpy()),
         )
 
+@frozen
+class _AttributeMemberMapping:
+    member_name: str
+    attribute_name: str = field()
```

Comment:
```suggestion
    attribute_name: str
```

Reply:
I need the field object to be able to call `attribute_name.default`.